diff --git a/Utilities/build-script-helper.py b/Utilities/build-script-helper.py
index 9892ffde..25d81608 100755
--- a/Utilities/build-script-helper.py
+++ b/Utilities/build-script-helper.py
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from __future__ import print_function
+
 
 import argparse
 from distutils import file_util
@@ -66,7 +66,7 @@ def swiftpm(action, swift_exec, swiftpm_args, env=None):
   subprocess.check_call(cmd, env=env)
 
 def swiftpm_bin_path(swift_exec, swiftpm_args, env=None):
-  swiftpm_args = list(filter(lambda arg: arg != '-v' and arg != '--verbose', swiftpm_args))
+  swiftpm_args = list([arg for arg in swiftpm_args if arg != '-v' and arg != '--verbose'])
   cmd = [swift_exec, 'build', '--show-bin-path'] + swiftpm_args
   print(' '.join(cmd))
   return subprocess.check_output(cmd, env=env).strip()
@@ -308,9 +308,8 @@ def install_libraries(args, build_dir, universal_lib_dir, toolchain_lib_dir, tar
       delete_rpath(driver_lib_dir_path, lib_path, args.verbose)
 
   # Fixup the TSC and llbuild rpaths
-  driver_libs = map(lambda d: os.path.join('lib', d), ['libSwiftDriver', 'libSwiftOptions', 'libSwiftDriverExecution'])
-  tsc_libs = map(lambda d: os.path.join('dependencies', 'swift-tools-support-core', 'lib', d),
-                 ['libTSCBasic', 'libTSCLibc', 'libTSCUtility'])
+  driver_libs = [os.path.join('lib', d) for d in ['libSwiftDriver', 'libSwiftOptions', 'libSwiftDriverExecution']]
+  tsc_libs = [os.path.join('dependencies', 'swift-tools-support-core', 'lib', d) for d in ['libTSCBasic', 'libTSCLibc', 'libTSCUtility']]
   for lib in driver_libs + tsc_libs:
     for target in targets:
       lib_path = os.path.join(build_dir, target,
