diff --git a/utils/build-presets.ini b/utils/build-presets.ini
index cbb45efe3fe..cdab357696e 100644
--- a/utils/build-presets.ini
+++ b/utils/build-presets.ini
@@ -1248,8 +1248,11 @@ swiftsyntax-verify-generated-files
 release-debuginfo
 compiler-vendor=apple
 
-# Cross compile for Apple Silicon
-cross-compile-hosts=macosx-arm64
+# Cross compile for Intel
+cross-compile-hosts=macosx-x86_64
+
+# Bump minimum deployment version
+darwin-deployment-version-osx=10.14
 
 lldb-use-system-debugserver
 lldb-build-type=Release
@@ -2648,25 +2651,25 @@ swift-install-components=autolink-driver;compiler;clang-builtin-headers;stdlib;s
 mixin-preset=source_compat_suite_macos_base
 debug
 assertions
-cross-compile-hosts=macosx-arm64
+cross-compile-hosts=macosx-x86_64
 
 [preset: source_compat_suite_macos_RA]
 mixin-preset=source_compat_suite_macos_base
 release
 assertions
-cross-compile-hosts=macosx-arm64
+cross-compile-hosts=macosx-x86_64
 
 [preset: source_compat_suite_macos_R]
 mixin-preset=source_compat_suite_macos_base
 release
 no-assertions
-cross-compile-hosts=macosx-arm64
+cross-compile-hosts=macosx-x86_64
 
 [preset: source_compat_suite_macos_D]
 mixin-preset=source_compat_suite_macos_base
 debug
 no-assertions
-cross-compile-hosts=macosx-arm64
+cross-compile-hosts=macosx-x86_64
 
 [preset: source_compat_suite_linux_DA]
 mixin-preset=source_compat_suite_linux_base
